.github/workflows/deploy-toolchain-action.yml

```diff
@@ -0,0 +1,55 @@
+name: Deploy Artifacts
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        required: false
+        type: string
+        description: Version to author
+    secrets:
+      SSHKEY:
+        required: true
+
+
+jobs:
+  upload_artifacts:
+    name: Deploy Artifacts ${{ matrix.arch }}-${{ matrix.platform }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        host:
+          - x86_64
+          - aarch64
+        target:
+          - x86_64
+          - aarch64
```

Comment:
Will we ever want host x86_64 and target aarch64?

Reply:
We already get that from this matrix.

Reply:
Yes, I know, the question was more if we really something different then a matching host to target(maybe I just don't understand why we need it)